### PR TITLE
Update Mininet patch for 2022 apr

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

**Comment by jafingerhut** [comment]
The latest version of Mininet has some changes to one of its install scripts, causing a patch that was being applied to that file to fail.  These changes update that patch, and cause it to be used for both `vagrant up` and `vagrant up dev` commands.

File: vm-ubuntu-20.04/user-common-bootstrap.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ set -xe
 git clone https://github.com/mininet/mininet mininet
```
Comment (rst0git) [inline vm-ubuntu-20.04/user-common-bootstrap.sh:7]: @jafingerhut would it be possible to clone a specific tag from the mininet repository?
For example:
```
MININET_COMMIT="2.3.0"
git clone -b ${MININET_COMMIT} https://github.com/mininet/mininet mininet
```

Comment (jafingerhut) [reply vm-ubuntu-20.04/user-common-bootstrap.sh:7]: Definitely possible.  At this time, there does not appear to be a Mininet tag version that corresponds to the 2022-Apr-01 change made to the util/install.sh file in the Mininet repo.  I'm happy to leave this PR open until there is such a tagged version of Mininet, and when there is, add such a tagged version to this PR before it is merged in.

In the mean time, I can create a separate PR that uses a tagged version with the current Mininet patch.

Comment (jafingerhut) [reply vm-ubuntu-20.04/user-common-bootstrap.sh:8]: The most recent tag for Mininet appears to be over 1 year old, so I added another commit to this PR that uses a particular recent commit of Mininet.  See what you think.

**Review by rst0git — APPROVED** [review]
LGTM

File: vm-ubuntu-20.04/user-common-bootstrap.sh (hunk 1 above)
Comment (rst0git) [reply vm-ubuntu-20.04/user-common-bootstrap.sh:8]: It looks good. Thank you for fixing this.

**Pull request merged**